cookbooks/aws-parallelcluster-environment/resources/lustre/partial/_install_lustre_centos_redhat.rb

```diff
@@ -28,17 +28,12 @@
     command "yum-config-manager --setopt=\*.skip_if_unavailable=1 --save"
   end
 
+  # TODO: restore installation on docker when Lustre is available for RH8.9
```

Comment:
Can't we force an 8.8 image for docker? Mmm. We are using the images from kitchen-dokken, so it looks like we can't. I wonder if we should start defining our own dokken images...